interface/main/tabs/menu/menus/standard.json

```diff
@@ -1230,22 +1230,39 @@
               "super"
             ],
             "global_req": [
-              "erx_enable",
-              "newcrop_user_role"
+              "erx_enable"
             ]
           },
           {
             "label":"Weno Management",
             "menu_id":"adm0",
             "target":"adm",
-            "url":"/interface/weno/admin.php",
+            "url":"/interface/weno/facilities.php",
             "children":[],
             "requirement":0,
             "acl_req": [
               "admin",
-              "super"
-            ]
-          }
+              "patients"
```

Comment:
note that 'admin' => 'patients' aco does not exist. 
recommend 'admin' => 'super' aco which you are using at the top of the facilities.php script